docs/standard/library-guidance/breaking-changes.md

```diff
@@ -0,0 +1,98 @@
+
+## Project types and breaking changes
+
+How a project is used by the .NET community changes the effect of breaking changes on end-user developers.
+
+* **Low and middle-level libraries** like a serializer, HTML parser, database object-relational mapper, or web framework are the most effected by breaking changes.
+
+  Building block packages are used by end-user developers to build applications, and by other projects as NuGet dependencies. For example, you're building an application and are using an open-source client to call a web service. A breaking update to dependency the client uses isn't something you can fix. It's the open-source client that needs to be changed and you have no control over it. You have to find compatible versions of the libraries, or submit a fix to the client library and wait for a new version. The worst-case situation is if you want to use two libraries that depend on mutually incompatible versions of a third library.
+
+* **High-level libraries** like a suite of UI controls are less sensitive to breaking changes.
+
+  High-level libraries are directly referenced in an end-user application. If breaking changes occur the developer can choose to update to the latest version, or can modify their application to work with the breaking change.
+
+**✔️ DO** think about how your library will be used. What effect will breaking changes have on applications and libraries that use it?
+
+**✔️ DO** minimize breaking changes when developing a low-level .NET library.
+
+**✔️ CONSIDER** publishing a major rewrite of a library as a new NuGet package.
+
+## Types of breaking changes
+
+Breaking changes fall into different categories and aren't equally impactful.
+
+### Source breaking change
+
+A source breaking change doesn't affect program execution but will cause compilation errors the next time the application is recompiled. For example, a new overload can create an ambiguity in method calls that were unambiguous previously, or a renamed parameter will break callers that use named parameters.
+
+```csharp
+public class Task
+{
+    // Adding a type called Task could conflict with System.Threading.Tasks.Task at compilation
+}
+```
+
+Because a source breaking change is only harmful when the developer recompiles their application, it's the least disruptive breaking change. Developers can fix their own broken source code easily.
+
+### Behavior breaking change
+
+Behavior changes are the most common breaking change: almost any change in behavior could break someone. Changing the results a method produces, the exception your library throws, or the data output it writes to the network or disk could all negatively impact applications and libraries that depend on you. Even a bug fix can qualify as a breaking change if users relied on the previously broken behavior.
```

Comment:
> Changing the results a method produces, the exception your library throws, or the data output it writes to the network or disk could all negatively impact applications and libraries that depend on you.

Incorrect articles, perhaps it could be reworded as: "_Changes to your library, such as method signatures, exceptions thrown or input or output data formats, could all negatively impact your library consumers_"?